app/controllers/datasets.js

```diff
@@ -17,7 +17,7 @@ export default Ember.Controller.extend({
 
   download_link: Ember.computed('model', 'model.years_available.@each.selected', function() {
     let yearsSelected = this.get('model.years_available') || [];
-    if (!yearsSelected.errors) {
+    if (Ember.compare(Ember.Error, yearsSelected) !== 0) {
```

Comment:
What is this? I've never compared an error to something before? Did you mean `yearsSelected instanceof Ember.Error`?